examples/monetization-with-stripe/pages/api/payment-handler.ts

```diff
@@ -0,0 +1,47 @@
+import { buffer } from "micro";
+import { storage } from "../../utils/storage";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import Stripe from "stripe";
+
+export const PAYMENT_STORAGE_KEY = "paid-for-[miro-integration-name]";
+
+const stripe = new Stripe(process.env.STRIPE_SECRET_KEY);
```

Comment:
Two small TypeScript errors that can be fixed:

```suggestion
const stripe = new Stripe(process.env.STRIPE_SECRET_KEY!, {apiVersion: '2022-11-15'});
```